.ci/java-versions.properties

```diff
@@ -5,6 +5,7 @@
 # are 'java' or 'openjdk' followed by the major release number.
 
 ES_BUILD_JAVA=openjdk12
-ES_RUNTIME_JAVA=java11
+ES_RUNTIME_JAVA=openjdk13
 GRADLE_TASK=build
+GRADLE_EXTRA_ARGS=-Dcompiler.java=$HOME/.java/openjdk13
```

Comment:
Note that this is temporary for testing this pull request. It will be removed before merging.